### PR TITLE
chore(developer): kmc-keyboard should not use any node-specific modules

### PR DESCRIPTION
**Pull request by mcdurdin**
One of a set of updates for kmc-*, as all of them currently have either wasm-fs or node core module references. This is the easiest patch.

@keymanapp-test-bot skip

### CODE REVIEW

**Review by darcywong00 — APPROVED** [review]
lgtm

File: common/web/types/src/main.ts [hunk 1]
```diff
@@ -28,3 +28,5 @@ export { TouchLayoutFileWriter, TouchLayoutFileWriterOptions } from './keyman-to
 export * as KPJ from './kpj/kpj-file.js';
 export { KPJFileReader } from './kpj/kpj-file-reader.js';
 export { KeymanDeveloperProject } from './kpj/keyman-developer-project.js';
+
+export * as util from './util/util.js';
```
Comment (srl295) [inline common/web/types/src/main.ts:32]: would be clearer as `fs-util.ts` or `path-util.ts` in light of the encapsulation mentioned.

Comment (mcdurdin) [reply common/web/types/src/main.ts:32]: Note that follow-up PR #8674 refactors the fs-related functions in util.ts away again, moving responsibility for path and fs ops into the callbacks (we may be able to do better later but avoiding adding extra deps for now). :grin:

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.96-alpha](https://keyman.com/downloads/releases/alpha/17.0.96)

**Pull request merged**